src/Tribe/Views/V2/View.php

```diff
@@ -1393,7 +1393,16 @@ protected function setup_template_vars() {
 			? Dates::build_date_object( $event_date )->format( Dates::DBDATEFORMAT )
 			: false;
 
-		$template_vars = [
+		/*
+		 * Some plugins, like WooCommerce, will modify the UID of logged out users; avoid that filtering here.
+		 *
+		 * @see TEC-3579
+		 */
+		$rest_nonce = tribe_without_filters( [ 'nonce_user_logged_out' ], static function () {
```

Comment:
* :no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call

<sub>hash: f2fad6b1238e0ade376c833916899bee</sub>